src/ProductConstructionService/ProductConstructionService.Api/Queue/QueueConfiguration.cs

```diff
@@ -14,7 +14,7 @@ public static void AddWorkitemQueues(this WebApplicationBuilder builder)
         var queueName = builder.Configuration[JobQueueConfigurationKey] ??
             throw new ArgumentException($"{JobQueueConfigurationKey} missing from the configuration");
 
-        builder.Services.AddSingleton<JobsProcessorStatus>();
+        builder.Services.AddSingleton(new JobsProcessorStatus(new ManualResetEventSlim(true)));
```

Comment:
Should we not make the ctor more user friendly? E.g. only make it accept a boolean with true/false whether we want to start as started or stopped? We shouldn't require the user of the class to pass some event manager set up in some way.